lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -301,3 +223,89 @@ class VersionList extends Component {
     </Panel>
   }
 }
+
+type ToolbarProps = ListProps & {size: string}
+
+export class VersionButtonToolbar extends Component<ToolbarProps> {
+  _onClickDownload = () =>
+    this.props.downloadFeedClicked(this.props.version, this.props.isPublic)
+
+  _onClickLoadIntoEditor = (evt: SyntheticMouseEvent<HTMLInputElement>) => {
+    const {loadFeedVersionForEditing, version} = this.props
+    const messages = getComponentMessages('FeedVersionViewer')
+    const {id: feedVersionId, feedSource} = version
+    this.refs.confirm.open({
+      title: getMessage(messages, 'load'),
+      body: getMessage(messages, 'confirmLoad'),
+      onConfirm: () =>
+        loadFeedVersionForEditing({feedSourceId: feedSource.id, feedVersionId})
+    })
+  }
+
+  _onClickDeleteVersion = (evt: SyntheticMouseEvent<HTMLInputElement>) => {
+    const {deleteFeedVersionConfirmed, version} = this.props
+    const messages = getComponentMessages('FeedVersionViewer')
```

Comment:
Move all these getComponentMessages to the root of this file.

Reply:
Hmm, actually I think `getComponentMessages` must be called within a class.  Otherwise, there's an issue accessing the config file attached to the window.

Reply:
I see.  Let's let this slide for now and if it begins to bother either of us in the future we'll revisit.